data/json/items/gunmod/magazine.json

```diff
@@ -12,13 +12,15 @@
     "symbol": ":",
     "color": "green",
     "location": "magazine",
-    "mod_targets": [ "rifle" ],
-    "acceptable_ammo": [ "223", "308" ],
+    "mod_targets": [ "rifle", "launcher" ],
+    "acceptable_ammo": [ "223", "308", "50", "40mm" ],
```

Comment:
Unless we have magazine fed grenade launchers there's nothing to use this mod then. Only define it there is an actual item that could make use of it. Unused definitions add complexity for no benefit.